components/global/Header/UserNav/index.js

```diff
@@ -1,21 +1,66 @@
+import PropTypes from "prop-types";
 import { useTranslation } from "react-i18next";
-import { useAuthModal } from "@/hooks";
+import { Popover, Portal } from "@headlessui/react";
+import { useAuthModal, useBoundingBox } from "@/hooks";
 import IconComposer from "@/svg/IconComposer";
 import { useAuthenticationContext } from "@/contexts/Authentication";
 import * as Styled from "./styles";
+import SRAuthStatus from "@/components/auth/SRAuthStatus";
 
-function UserNav() {
+function UserNav({ headerVisible }) {
   const { openModal } = useAuthModal();
   const { t } = useTranslation();
-  const { isAuthenticated } = useAuthenticationContext();
+  const { isAuthenticated, signOut } = useAuthenticationContext();
+  const [box, ref] = useBoundingBox();
 
   return (
-    <Styled.Nav aria-label="User">
+    <Styled.Nav
+      aria-label="User"
+      aria-live="polite"
+      className="c-nav-list--desktop"
+      ref={ref}
+    >
+      <SRAuthStatus />
```

Comment:
@dananjohnson I wasn't quite sure where to put the auth status, since the location of the sign in / sign out buttons change on desktop and mobile. So I opted to add it near the buttons - here it is on the user nav. I might be over thinking this - would it be better to just have the status sit in the header?

Reply:
Maybe higher in the header? I could see us using it for other events too.

Btw, `role="status"` has implicit `aria-live="polite"`, so you don't need to add that to the parent nav. Unless there's some other intent I'm missing?

Reply:
Oh, sweet! I can leave the aria-live status around the user nav if we want to announce the button changes, but if we just want to announce the status I'll move it higher in the header.

Reply:
I think just announcing the status is fine. Announcing "Log in" after log out might be confusing without more context.

Reply:
@dananjohnson Cool cool. I've removed the extra aria-live regions and bumped the SRAuth announcement to the header. We can always extend it later to a more generic component if we need to. There's also the HOC (that I didn't see until now, of course) called `withLiveRegionChange`. If it looks good, approve it and I'll merge it in!

Reply:
Oh right, I forgot about the HOC. 🤷🏻 Sounds good though!